model/ruleset.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+from typing import Iterable
+from model.rules import Rule
+from model.scopes_conditions import ScopeCondition
+
+
+@dataclass(frozen=True)
+class ScopedRuleSet:
+    """Set of rules to be applied to matching set of scope conditions, depending on the matched scope action."""
+
+    scope: Iterable[ScopeCondition]
+    """A combination of scope conditions which defines the scope as a whole. The first scope condition that gets matched defines the outcome of whether rules should be evaluated or not, therefore scope condition order is relevant."""
+    rule_set: Iterable[Rule]
```

Comment:
"ruleset" is acceptable as one word and will prevent possible confusion in the future.
Same for the class name.

Reply:
Also changed in README